pkgs/by-name/or/ory-hydra/package.nix

```
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  buildGoModule,
+  lib,
+}:
+buildGoModule rec {
+  pname = "ory-hydra";
+  version = "2.3.0";
+  commit = "ee8c339ddada3a42529c0416897abc32bad03bbb";
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "hydra";
+    rev = "v${version}";
+    hash = "sha256-f/pBRrFMfpcYSfejIGpCD5Kywtg5oyovw5RemvRDPTs=";
+  };
+
+  vendorHash = "sha256-g2NDPwLgM/LmndCgh5pXjc1DJ3pnGcHlWm+opPVK1bE=";
+
+  subPackages = [ "." ];
+
+  # Pass versioning information via ldflags
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/ory/hydra/v2/driver/config.Version=${version}"
+    "-X github.com/ory/hydra/v2/driver/config.Commit=${commit}"
```

```suggestion
    "-X github.com/ory/hydra/v2/driver/config.Commit=${src.tag}"
```
just mock that. Otherwise we need a custom updater which is not worth the extra complexity

---

I had to use src.rev instead of src.tag